boards/arm/lpc43xx/bambino-200e/configs/netnsh/Make.defs

```diff
@@ -50,7 +50,7 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
   ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
 endif
 
-ARCHCFLAGS = -fno-builtin
+ARCHCFLAGS = -fno-common -fno-builtin
 ifeq ($(CONFIG_CXX_EXCEPTION),y)
   ARCHCPUFLAGSXX = -fno-builtin
```

Comment:
```suggestion
  ARCHCPUFLAGSXX = -fno-common -fno-builtin
```
and for `else` case

Reply:
Remove from https://github.com/apache/incubator-nuttx/pull/5555